hydra-main/src/main/java/com/addthis/hydra/query/web/HttpQueryHandler.java

```diff
@@ -284,6 +284,32 @@ private void fastHandle(ChannelHandlerContext ctx, FullHttpRequest request, Stri
                 writer.write(q.getPaths()[0]);
                 break;
             }
+            case "/mqmaster/deactivate": {
+                log.trace("Received MeshQueryMaster deactivate request");
+                String shouldDeactivate = kv.getValue("deactivate");
+                boolean success;
+                switch (shouldDeactivate) {
+                    case "1":   success = meshQueryMaster.deactivateMqMaster(true);
+                        break;
+                    case "0":   success = meshQueryMaster.deactivateMqMaster(false);
+                        break;
+                    default:  success = false;
+                }
+                if(success) {
+                    writer.write("[1]");
+                } else {
+                    writer.write("[0]");
+                }
+                break;
+            }
```

Comment:
We should just set deactivate to true or false, and not worry about sending back success or state. If we get an http 200, assume all is well.

Reply:
Edited deactivate endpoint to just set() and return Http ok/400